### PR TITLE
Fix sonar issues

### PR DESCRIPTION
**Pull request by mikerasch**
Fixes sonar issues by adding documentation and changing access modifiers to private

### CODE REVIEW

File: application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildLeaveCloseThreadListener.java [hunk 1]
```diff
@@ -23,6 +23,11 @@ public class OnGuildLeaveCloseThreadListener extends ListenerAdapter implements
             LoggerFactory.getLogger(OnGuildLeaveCloseThreadListener.class);
     private final Database database;
 
+    /**
+     * Creates a new instance
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildLeaveCloseThreadListener.java:27]: missing dot.

File: application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildLeaveCloseThreadListener.java [hunk 2]
```diff
@@ -23,6 +23,11 @@ public class OnGuildLeaveCloseThreadListener extends ListenerAdapter implements
...
+     *
+     * @param database database to use
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/help/OnGuildLeaveCloseThreadListener.java:29]: use for what?

File: application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadMetadataPurger.java [hunk 3]
```diff
@@ -19,6 +19,11 @@ public class HelpThreadMetadataPurger implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadMetadataPurger.class);
     private static final Period DELETE_MESSAGE_RECORDS_AFTER = Period.ofDays(30);
 
+    /**
+     * Creates a new instance
+     *
+     * @param database the database to use
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadMetadataPurger.java:25]: use for what?

File: application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadMetadataPurger.java [hunk 4]
```diff
@@ -19,6 +19,11 @@ public class HelpThreadMetadataPurger implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadMetadataPurger.class);
     private static final Period DELETE_MESSAGE_RECORDS_AFTER = Period.ofDays(30);
 
+    /**
+     * Creates a new instance
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadMetadataPurger.java:23]: missing dot.

File: application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java [hunk 5]
```diff
@@ -56,6 +56,7 @@ public final class HelpSystemHelper {
      * Creates a new instance.
      *
      * @param config the config to use
+     * @param database the database to use
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java:59]: use for what?

**Review by Zabuzard — CHANGES_REQUESTED** [review]
Good stuff. But I couldnt just accept it without requesting _something_ 😄

**Review by Zabuzard — APPROVED** [review]

**Review by marko-radosavljevic — APPROVED** [review]

**Pull request merged**